amqp_influxdb/__init__.py

```diff
@@ -44,6 +44,10 @@ def __init__(self,
 
         connection_parameters = connection_parameters or {}
 
+        if 'credentials' not in connection_parameters:
```

Comment:
Um.. I don't quite understand the logic behind this. having `connection_parameters = connection_parameters or {}` was meant to provide a way for someone to initialize the class without these parameters.
If we send the creds both from the initiator (`__main__.py`) and hardcode them into the properties of the params, what did we achieve?